### DISCUSSION TITLE
Showcasing my new project: "web-authn-completed-app"

### DISCUSSION THREAD

**Discussion by guMcrey** [MasterKale/SimpleWebAuthn#355] [Show and tell]
### [web-authn-completed-app](https://github.com/guMcrey/web-authn-completed-app)

Hello everyone, 

I would like to share with you my latest project - [web-authn-completed-app](https://web-authn.x-dev.club). This application demonstrates the capabilities of SimpleWebAuthn by providing a real-world example of how the technology can be integrated into a website.

The readme file provides a detailed explanation of the project and its features, including a step-by-step guide to set up and run the app. Additionally, the project has been thoroughly tested and is fully functional, making it a reliable resource for those looking to integrate SimpleWebAuthn into their websites.

**💻 [online preview](https://web-authn.x-dev.club)**

### ✨ Feature
- Clean and modern user interface
- Support docker deployment
- With its clean code, detailed documentation, MIT License

**[GIF demo](https://web-authn.x-dev.club)**

<img width="55%" src="https://user-images.githubusercontent.com/26329117/218042751-04df521c-a87a-4231-8faa-c8ba53d9b198.gif" />

---

I hope you like it! I look forward to having the community review our code and welcome any questions or feedback you may have.  

Best regards,

[guMcrey(hakuna matata)](https://github.com/guMcrey)

**Comment by MasterKale** [comment]
Hello @guMcrey, thank you for sharing, this looks great! I'd love to hear about how you decided to create this. And is it a portfolio piece? Are you planning on keeping it up-to-date with passkey "best practices" as they come out?

**Comment by guMcrey** [comment]
Hello! I'm glad you like it 😊.

There are two reasons for writing this project:

1. Initially, it was a research task at work, where I learned about passkey, and after researching a lot of materials, I found it very interesting and powerful. So I wanted to make a more complete demo to help me understand how it integrates with websites and works. During the learning and understanding process, I felt that it is relatively high-threshold for most developers, especially in China, so I added some operation videos and multiple languages to my website, hoping that through my project, friends who are interested in passkey can know more easily what it is and the source code can also help everyone get started faster.

2. It also serves as a personal project to showcase my skills in website development and design. I want to create a user-friendly, responsive, and visually appealing web application to prove my ability to use modern web technologies.

The project is still being improved and optimized, and I am very willing to continue to follow the latest features and best practices of passkey, and update them to my project accordingly.

I would also like to ask if it's convenient to add my [Github project address](https://github.com/guMcrey/web-authn-completed-app) to the SimpleWebAuthn documentation or website, so that more people can follow it.

Thank you very much.
